ios/FluentUI.Demo/FluentUI.Demo/Demos/TooltipDemoController.swift

```diff
@@ -20,20 +20,23 @@ class TooltipDemoController: DemoController {
         navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Show on title", style: .plain, target: self, action: #selector(showTitleTooltip))
 
         container.addArrangedSubview(createButton(title: "Show single-line tooltip below", action: { sender in
-            Tooltip.shared.show(with: "This is pointing up.", for: sender, preferredArrowDirection: .up)
+            Tooltip.shared.show(with: "This is pointing up.", title: nil, for: sender, preferredArrowDirection: .up)
```

Comment:
why doe did you need to update the democontroller calls if it is just nil?

Reply:
just to get rid of the deprecation warnings, should I leave it?

Reply:
should there be 3 different show() functions now?